### PR TITLE
Fix/goal kick outside

### PR DESCRIPTION
**Pull request by PhiBabin**
- Fix bug where a area in front of goal was impossible to kick.
- Add an area forbidden behind goal
- Fix a bug in defend_wall

### CODE REVIEW

File: ai/STA/Tactic/go_to_position.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from typing import List
 
+from Debug.debug_command_factory import DebugCommandFactory, VIOLET
```
Comment (sim0504) [inline ai/STA/Tactic/go_to_position.py]: ? Violet it is

File: ai/STA/Tactic/tactic.py [hunk 2]
```diff
@@ -109,12 +118,18 @@ def _find_best_next_target(self, area: Area, old_target_position, player_positio
                 return intersections[1]
 
     def debug_cmd(self):
-        #return [DebugCommandFactory().area(area) for area in self.forbidden_areas]
-        return []
+        cmds = []
+        [cmds.extend(DebugCommandFactory().area(area, color=VIOLET)) for area in self.forbidden_areas]
```
Comment (sim0504) [inline ai/STA/Tactic/tactic.py]: The list should be inside the extend method. it can take an iterable to extend the list.

Comment (PhiBabin) [reply ai/STA/Tactic/tactic.py]: Does not work

Comment (PhiBabin) [reply ai/STA/Tactic/tactic.py]: Literable est la liste de ce qui est extend, pas la liste de liste

Comment (sim0504) [reply ai/STA/Tactic/tactic.py]: `cmds.extend([DebugCommandFactory().area(area, color=VIOLET) for area in self.forbidden_areas])` ?

Comment (sim0504) [reply ai/STA/Tactic/tactic.py]: Ah je comprends

**Pull request merged**